webapp/lib/controller.dart

```diff
@@ -120,24 +118,24 @@ void loadProjectSelectorView() {
 void loadDashboardView() {
   view.navView.projectTitle = project?.projectName;
   view.navView.projectOrganizations = [''];
-  var dashboardView = new view.DashboardView();
+  var dashboardView = new view.DashboardView(model.conversationData);
   dashboardView.activePackages.addAll(
     [
-      new view.ActivePackagesViewPartial('Urgent conversations', '#/conversations', '#/escalates-configuration'),
-      new view.ActivePackagesViewPartial('Open conversations', '#/conversations', '#'),
-      new view.ActivePackagesViewPartial('Batch replies (Week 12)', '', '#/batch-replies-configuration'),
+      new view.ActivePackagesViewPartial('Urgent conversations', '#/conversations', '#/escalates-configuration', '3 awaiting reply'),
```

Comment:
can we get the number `3` from `conversationData['needs-urgent-intervention']`?

Reply:
Okay. Changed